tests/integration/test_api_gateway.py

```diff
@@ -798,6 +800,87 @@ def test_import_rest_api(self):
         # clean up
         client.delete_rest_api(restApiId=rest_api_id)
 
+    def test_step_function_integrations(self):
+        client = aws_stack.connect_to_service('apigateway')
+        sfn_client = aws_stack.connect_to_service('stepfunctions')
+        lambda_client = aws_stack.connect_to_service('lambda')
+
+        state_machine_name = 'test'
+        state_machine_def = {
+            'Comment': 'Hello World example',
+            'StartAt': 'step1',
+            'States': {
+                'step1': {
+                    'Type': 'Task',
+                    'Resource': '__tbd__',
+                    'End': True
+                },
+            }
+        }
+
+        fn_name = 'test-stepfunctions-apigw'
+
+        # create state machine
+        testutil.create_lambda_function(
+            handler_file=TEST_LAMBDA_ECHO_FILE,
+            func_name=fn_name,
+            runtime=LAMBDA_RUNTIME_PYTHON36
+        )
+
+        role_arn = aws_stack.role_arn('sfn_role')
+
+        definition = clone(state_machine_def)
+        lambda_arn_1 = aws_stack.lambda_function_arn(fn_name)
+        definition['States']['step1']['Resource'] = lambda_arn_1
+        definition = json.dumps(definition)
+        sm_arn = 'arn:aws:states:us-east-1:%s:stateMachine:%s' \
```

Comment:
nit: Let's better parameterize the region here as well, using `aws_stack.get_region()` (instead of hardcoding `us-east-1`)